src/core.ts

```diff
@@ -117,8 +117,8 @@ export function instrument(eventState: IInstrumentationEventState) {
 }
 
 function getUnixTimestamp(): number {
-  return (window.performance && window.performance.now)
```

Comment:
There are some cases when performance object exists, without the 'now' method, but it always has the 'timing' property